source/openpulse/tests/test_openqasm_tests.py

```diff
@@ -0,0 +1,13 @@
+# Copied from openqasm3. OpenPulse parser should be able to parse openqasm3 examples
+import openpulse
+
+
+def test_examples(example_file):
```

Comment:
Is this parameterized as a separate pytest test for each test file?

Reply:
Yes - `pytest` has a weird (to me, at least) style where tests that take an argument implicitly look for things in pytest's scope that are decorated as `@pytest.fixture`.  The `example_file` fixture is in `conftest.py`, and it's parametrised to list each example file (except for excludes).